packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -382,15 +399,44 @@ export class ChildLogger extends TelemetryLogger {
 		if (loggerIsMonitoringContext(baseLogger)) {
 			mixinMonitoringContext(this, new CachedConfigProvider(this, baseLogger.config));
 		}
+		this.baseLogger = new FilteredLogger(baseLogger, undefined, undefined);
 	}
 
 	/**
 	 * Send an event with the logger
 	 *
 	 * @param event - the event to send
 	 */
-	public send(event: ITelemetryBaseEvent): void {
-		this.baseLogger.send(this.prepareEvent(event));
+	public send(event: ITelemetryBaseEvent, logLevel?: LogLevel): void {
+		this.baseLogger.send(this.prepareEvent(event), logLevel);
```

Comment:
ideally, we would want filtering before enrichment, specifically prepareEvent. i don't yet have a clear picture of how to do that.